src/components/button/button.vue

```diff
@@ -10,86 +10,109 @@ module.exports = {
     }
     &:focus {
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px var(--primary-disabled);
+      box-shadow: 0 0 0 1px var(--neutral-seven), 0 0 0 3px var(--neutral-six);
       outline: none;
     }
     &[disabled] {
-      background-color: var(--primary-disabled);
-    }
-    &[disabled]:hover {
-      cursor: not-allowed;
+      background-color: var(--neutral-nine);
+      color: var(--neutral-five);
+      border: 1px solid var(--neutral-eight);
+      &:hover {
+        cursor: not-allowed;
+      }
     }
   }
 
-  .apos-button--secondary {
-    background-color: var(--secondary);
+  .apos-button--primary {
+    color: var(--white);
+    background: var(--primary);
+    border: none;
     &:hover {
-      background-color: var(--secondary-hover);
+      background-color: var(--primary-button-hover);
     }
     &:active {
-      background-color: var(--secondary-active);
+      background-color: var(--primary-button-active);
     }
     &:focus {
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px var(--secondary-disabled);
+      box-shadow: 0 0 0 1px var(--neutral-seven), 0 0 0 3px var(--primary-button-disabled);
     }
     &[disabled] {
-      background-color: var(--secondary-disabled);
+      background-color: var(--primary-button-disabled);
+      color: var(--white);
+      border: none;
     }
   }
 
-  .apos-button--danger {
-    background-color: var(--danger);
+  .apos-button--input {
+    background-color: var(--neutral-two);
+    color: var(--neutral-ten);
+    border-color: var(--neutral-four);
     &:hover {
-      background-color: var(--danger-hover);
+      background-color: var(--neutral-one);
     }
     &:active {
-      background-color: var(--danger-active);
+      background-color: var(--neutral-one);
     }
     &:focus {
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px var(--danger-disabled);
+      box-shadow: 0 0 0 1px var(--neutral-seven), 0 0 0 3px var(--neutral-one);
     }
     &[disabled] {
-      background-color: var(--danger-disabled);
+      background-color: var(--neutral-four);
+      color: var(--neutral-seven);
     }
   }
 
-  .apos-button--cancel {
-    background-color: var(--cancel);
-    color: var(--text-primary);
-    border: 1px solid var(--neutral-six);
+  .apos-button--danger {
+    color: var(--white);
+    background-color: var(--danger);
+    border: none;
     &:hover {
-      background-color: var(--cancel-hover);
+      background-color: var(--danger-button-hover);
     }
     &:active {
-      background-color: var(--cancel-active);
+      background-color: var(--danger-button-active);
     }
     &:focus {
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px var(--cancel-hover);
+      box-shadow: 0 0 0 1px var(--neutral-seven), 0 0 0 3px var(--danger-button-disabled);
     }
     &[disabled] {
-      background-color: var(--cancel-disabled);
-      color: var(--neutral-three)
+      color: var(--white);
+      background-color: var(--danger-button-disabled);
```

Comment:
Yeah, this is bouncing back and forth between semantic variables and not. I think we commit to the semantic vars. It makes the vars list longer, but theme switching should be easier.